### ISSUE TITLE
Super Accurate Shotgun Ammo

### ISSUE THREAD

**Issue by PhiloTFarnsworth** [CleverRaven/Cataclysm-DDA#50663]
### Describe the bug

There are several 'families' of ammo which have a zero dispersion value because they inherit their properties from base ammos with unset dispersion values.  Many ammo variants make use of the relative dispersion values and make reference in flavor text to being less accurate versions of their parent, but are functionally just as accurate. 

A non-exhaustive list of affected ammos:
00-shot
.410 000 shot
20x66mm

### Steps To Reproduce

1. Compare 00-shot and 00-shot, scrap loaded values.

### Expected behavior

00-shot and 00-shot, scrap loaded should have different dispersion values.

### Versions and configuration

- OS: Windows 10
- Game Version: ed2f630
- Graphics version: Tiles
- Ingame language: English
- Mods loaded: None (Started game through play now)

**Comment by actual-nh** [comment]
Perhaps a warning when a proportional modifier is not doing anything (because it's multiplying 0) would also be a good idea? See `shot_scrap` for one instance of this.

**Comment by PhiloTFarnsworth** [comment]
I would agree on that.  I might even suggest that it be applied to any default value that is a proportionally or relatively modified.  While there's many instances where default values are correctly modified, from a best practices point of a view, it might be better to ensure that values that are inherited in the JSON are always explicitly set in the parent object.  This would further the goal of readability of the JSON, but it might also be a lot of work for relatively little gain.

**Comment by Marrim** [comment]
Due to the nature of shot-type ammo, it doesn't really make sense for it to have a "dispersion" of anything other than zero.
If a shotgun has 40-inch spread at 40 yards, having a dispersion above zero would mean the entire shot can veer more than 40 inches to a side and miss, even with perfect aim!

Even poor-quality is not that terribly inaccurate. If anything, scrap shot would translate to wider spread.
